example/home/blocks.py

```diff
@@ -89,6 +92,24 @@ class TextAndButtonsBlock(blocks.StructBlock):
     ]
 
 
+@register_streamfield_block
+class TextWithCallableBlock(blocks.StructBlock):
+    text = blocks.CharBlock()
+
+    graphql_fields = [
+        GraphQLString("text"),
+        GraphQLField("string_property", graphene.String, source="get_string_property"),
```

Comment:
@kbayliss have you looked at the followin use case:

```python
class TextWithCallableBlock(blocks.StructBlock):
    text = blocks.CharBlock()

    graphql_fields = [
        GraphQLString("text"),
        GraphQLString("label"),
    ]

    @property
    def label(self):
        return "my block property"
```

? it would make it consistent with

```python
class MyPage(Page)
    #...
    graphql_fields = [
        GraphQLString("label"),
    ]

    @property
    def label(self):
        return "Event"
```

which "just works"